### PR TITLE
Rename repo.url -> tufrepo.url

### PR DESCRIPTION
**Pull request by OYTIS**
Also pass this URL to garage-sign.

Waiting for targets keys format being fixed to test that.

### CODE REVIEW

File: classes/image_types_ostree.bbclass [hunk 1]
```diff
@@ -198,11 +198,7 @@ IMAGE_CMD_garagesign () {
             garage-sign init --repo ${GARAGE_SIGN_REPO} --home-dir ${GARAGE_SIGN_REPO} --credentials ${SOTA_PACKED_CREDENTIALS}
         fi
 
-        if [ -n "${GARAGE_SIGN_REPOSERVER}" ]; then
-            reposerver_args="--reposerver ${GARAGE_SIGN_REPOSERVER}"
-        else
-            reposerver_args=""
-        fi
+        reposerver_args="--reposerver $( unzip -p ${SOTA_PACKED_CREDENTIALS} tufrepo.url )"
```
Comment (patrickvacek) [inline classes/image_types_ostree.bbclass]: Do you need to cat that URL or can we just use the file as is?

Comment (OYTIS) [reply classes/image_types_ostree.bbclass]: That is the interface as was described by Simao. We can ask @simao or @jochenschneider to process tufrepo.url internally instead, but it doesn't work yet to my understanding.

File: classes/image_types_ostree.bbclass [hunk 2]
```diff
@@ -183,7 +183,7 @@ IMAGE_DEPENDS_garagesign = "garage-sign-native:do_populate_sysroot"
 IMAGE_CMD_garagesign () {
     if [ -n "${SOTA_PACKED_CREDENTIALS}" ]; then
         # if credentials are issued by a server that doesn't support offline signing, exit silently
-        unzip -p ${SOTA_PACKED_CREDENTIALS} root.json targets.pub targets.sec repo.url 2>&1 >/dev/null || exit 0
+        unzip -p ${SOTA_PACKED_CREDENTIALS} root.json targets.pub targets.sec turrepo.url 2>&1 >/dev/null || exit 0
```
Comment (jochenschneider) [inline classes/image_types_ostree.bbclass]: Spelling: "turrepo.url" -> "tufrepo.url"

File: classes/image_types_ostree.bbclass (hunk 1 above)
Comment (patrickvacek) [reply classes/image_types_ostree.bbclass]: No problem, just checking.

File: classes/image_types_ostree.bbclass (hunk 2 above)
Comment (OYTIS) [reply classes/image_types_ostree.bbclass]: Thanks!

**Pull request merged**